openapi/controllers/pinguin.py

```diff
@@ -669,16 +669,9 @@ def wrap__resource__create_one(modelname, context, data, success_code, out_field
               otherwise error response
     :rtype: werkzeug.wrappers.Response
     """
-    model_obj = get_model_for_read(modelname)
+    model_obj = get_model(modelname)
     try:
         created_obj = model_obj.with_context(context).create(data)
-        test_mode = request.registry.test_cr
-        if not test_mode:
-            # Somehow don't making a commit here may lead to error
-            # "Record does not exist or has been deleted"
-            # Probably, Odoo (10.0 at least) uses different cursors
-            # to create and to read fields from database
-            request.env.cr.commit()
```

Comment:
Without this, computed fields in odoo 13 don't work. Though ``commit`` can be replaced with ``flush`` there